tests/_support/Helper/Acceptance/ConvertKitAPI.php

```diff
@@ -222,7 +222,7 @@ public function apiUnsubscribe($id)
 	public function apiCustomFieldDataIsValid($I, $subscriber, $excludeNameFromAddress = false)
 	{
 		$I->assertEquals($subscriber['fields']['last_name'], 'Last');
-		$I->assertEquals($subscriber['fields']['phone_number'], '123-123-1234');
```

Comment:
WooCommerce [now strips ](https://github.com/woocommerce/woocommerce/pull/52476) all characters that are non-numeric or not a `+` symbol.